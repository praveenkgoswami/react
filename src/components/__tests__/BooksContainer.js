import React from 'react';
import { shallow } from 'enzyme';
import BooksContainer from '../books_container';

describe("BooksContainer", () => {
  it("Should render correctly", () => {
    expect(shallow(<BooksContainer />)).toMatchSnapshot();
  })
})
