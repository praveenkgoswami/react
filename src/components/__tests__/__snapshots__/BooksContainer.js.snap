// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BooksContainer Should render correctly 1`] = `
<div
  className="books-container"
>
  <h2
    className="text-center"
  >
    Books List
  </h2>
  <Connect(BookList) />
</div>
`;
